agreement_legal/models/agreement.py

```diff
@@ -0,0 +1,458 @@
+# Copyright (C) 2018 - TODAY, Pavlov Media
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class Agreement(models.Model):
+    _name = "agreement"
```

Comment:
@ygol Please remove this line

Reply:
it doesn't seem to work without "_name". Would you double check this?

Reply:
@ygol Use _inherit = "agreement" because Agreement class is already created in agreement module.

Reply:
@murtuzasaleh what do you mean?!? 
It is there https://github.com/ygol/contract/blob/eaa3bc947880699bad9f5bd82a9a5634a524a906/agreement_legal/models/agreement.py#L9